src/Modal/_ModalDialog.scss

```diff
@@ -230,22 +230,6 @@
 
 // Color Variants
 
-.pgn__modal-default {
-  // Default style modals don't have a background on the header which
-  // ends up looking spaced too far away from the body content.
-  .pgn__modal-header {
-    padding-bottom: calc(#{$modal-inner-padding} / 2);
-  }
```

Comment:
Are you sure it's ok to remove this? I agree with the comment here that without these styles there is too much space between header and the body

![image](https://github.com/openedx/paragon/assets/52399399/835f9319-8eeb-40bb-bed9-156b35b3366f)

can we instead move these styles to `.pgn__modal-header` so it's applied to every header?